samples: bluetooth: fast_pair: input_device: Update PN testing steps doc (#23736)
nrfconnect/sdk-nrf: samples/bluetooth/fast_pair/input_device/README.rst

Testing Personalized Name extension is described in `Fast Pair Certification Guidelines for Personalized Name`_.

.. note::

>    For Android devices running Android 15 or lower, to mitigate Android Personalized Name write issues, whenever you change the Personalized Name on an Android phone, perform the following:

Reviewer: ```suggestion
   For Android devices running Android 15 or lower, to mitigate Android Personalized Name write issues when you change the Personalized Name, perform the following:
```

>    * Write the new Personalized Name.

Reviewer: Aren't these steps that should be performed in this order? If yes, use a numbered list.
